github/issues_events.go

```diff
@@ -156,3 +161,15 @@ type Rename struct {
 func (r Rename) String() string {
 	return Stringify(r)
 }
+
+// DismissedReview contains details for 'dismissed_review' events
+type DismissedReview struct {
+	State             *string `json:"state,omitempty"`
```

Comment:
Please enumerate the possible values of `state` here, found at:
https://developer.github.com/v3/issues/events/#attributes

Reply:
I can see this being done in the code by via comments:

https://github.com/google/go-github/blob/d7732128a00e8e95e8fe896017da18ee20b2180d/github/checks.go#L139

I am wondering if you think this would suffice or whether we should add a custom `UnmarshalJSON` method here?

Reply:
Hmmm... I'm not sure I understand what you are saying here.

I see that line in another file, but I'm feeling that that is unrelated... and I'm not sure why we are talking about `UnmarshalJSON` here at all.

I was just asking for a comment for the `State` field... something like this:

```go
// State represents the state of the dismissed review and can be one of: ...
State             *string `json:"state,omitempty"`
```

Reply:
> I was just asking for a comment for the State field... something like this

Sorry yes, I thought this is what you meant, this was the reasoning for my snippet. Although it wasn't clear, the comment was what I was trying to communicate with the snipped 😆.

> and I'm not sure why we are talking about UnmarshalJSON here at all

This was due to my misunderstanding, I wasn't sure if you wanted a comment like the above or we wanted to say that the field can be one of X values, and so we would need to permit those values instead of allowing any value to come back.

Anyway, it's clear that we just need a comment here so I will add. Thanks 👍